js/id/renderer/background_source.js

```diff
@@ -4,6 +4,7 @@ iD.BackgroundSource = function(data) {
         name = source.name;
 
     source.scaleExtent = data.scaleExtent || [0, 20];
+    source.overzoom = data.overzoom !== false ? true : false;
```

Comment:
No need for a ternary here; `data.overzoom !== false` is already a boolean value.